tvaultuiv2/src/views/private/safe/components/SafeDashboard/index.js

```diff
@@ -337,7 +345,8 @@ const SafeDashboard = () => {
   const onSelectChange = (value) => {
     setSafeType(value);
     if (value !== 'All Safes') {
-      const obj = selectList.find((item) => item.selected === value);
+      const obj = selectList?.find((item) => item.selected === value);
```

Comment:
If selectList is undefined, obj will be assigned a boolean value - false - and hence the below code will break setSafeList([...safes[**obj.path**]]);

Please handle it accordingly

Reply:
selectList is initial state which is not depending on any API data

Reply:
it wont be undefined